src/lexer.jl

```diff
@@ -298,7 +298,7 @@ takechar(io::IO) = (readchar(io); io)
 
 #= Token Stream =#
 
-typealias Token Union(Symbol, Char, Number, Nothing)
+typealias Token Union{Symbol, Char, Number, Void}
```

Comment:
as indicated by the travis failure, this is broken on julia 0.3, and should not have been merged without `@compat`